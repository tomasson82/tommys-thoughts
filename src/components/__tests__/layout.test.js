import React from "react"
import renderer from "react-test-renderer"

import Layout from "../layout"

describe("Layout - test jest setup", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Layout location={{ pathname: "/" }} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
