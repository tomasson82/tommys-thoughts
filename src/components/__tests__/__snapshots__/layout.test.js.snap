// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout - test jest setup renders correctly 1`] = `
<div
  style={
    Object {
      "marginLeft": "auto",
      "marginRight": "auto",
      "maxWidth": "42rem",
      "padding": "2.625rem 1.3125rem",
    }
  }
>
  <header>
    <h1
      style={
        Object {
          "fontSize": "3.95285rem",
          "lineHeight": "4.375rem",
          "marginBottom": "2.625rem",
          "marginTop": 0,
        }
      }
    >
      <a
        href="/"
        style={
          Object {
            "boxShadow": "none",
            "color": "inherit",
          }
        }
      />
    </h1>
  </header>
  <main />
  <footer>
    Â© 
    2020
    , Built with
     
    <a
      href="https://www.gatsbyjs.org"
    >
      Gatsby
    </a>
  </footer>
</div>
`;
